test/graphql.createOrder.test.js

```diff
@@ -381,8 +392,10 @@ describe('createOrder', () => {
       expect(res.errors).to.not.exist;
       // Then the created transaction should match the requested data
       const orderCreated = res.data.createOrder;
+      expect(orderCreated.status).to.equal('ACTIVE');
```

Comment:
can you also add tests for the other possible statuses (CANCELED, REJECTED, ERROR)?